website/static/style.css

```
@@ -95,11 +117,32 @@ body {
 }
 
 .flair, a.flair:link, a.flair:visited  {
-    padding: 1px;
     border: solid;
-    border-width: 1px;
-    color: black;
-    border-color: black;
-    background-color: white;
-    font-weight:400;
+    border-width: 0;
```

Ici aussi, j'aimais bien la bordure pour la lisibilité:
```suggestion
    border-color: #a6a6a6;
    border-width: 1px;
}
```